src/Auth/Eloquent/UserQueryBuilder.php

```diff
@@ -14,4 +14,12 @@ protected function transform($items, $columns = ['*'])
             return User::make()->model($model);
         });
     }
+
+    protected function column($column)
+    {
+        return match ($column) {
+            'id' => User::make()->model()->getKeyName(),
+            default => $column,
+        };
+    }
```

Comment:
Please use an `if` guard for the id. We can't use the `match` syntax as we still support 7.4.

Reply:
Done!